R_datasets.Rmd

```diff
@@ -0,0 +1,8 @@
+# R data sets
```

Comment:
File name should only have lower case letters, i.e. r_datasets.Rmd - otherwise they don't sort nicely.

Reply:
already changed